src/mariadb.py

```diff
@@ -0,0 +1,224 @@
+  "10.0.26": "2016-06-23",
+  "10.0.25": "2016-04-28",
+  "10.0.24": "2016-02-17",
+  "10.0.23": "2015-12-16",
+  "10.0.22": "2015-10-28",
+  "10.0.21": "2015-08-05",
+  "10.0.20": "2015-06-17",
+  "10.0.19": "2015-05-08",
+  "10.0.18": "2015-05-06",
+  "10.0.17": "2015-02-25",
+  "10.0.16": "2015-01-25",
+  "10.0.15": "2014-11-21",
+  "10.0.14": "2014-09-24",
+  "10.0.13": "2014-08-08",
+  "10.0.12": "2014-06-12",
+  "5.5.68": "2020-05-06",
+  "5.5.67": "2020-01-18",
+  "5.5.66": "2019-10-30",
+  "5.5.65": "2019-07-25",
+  "5.5.64": "2019-04-26",
+  "5.5.63": "2019-01-27",
+  "5.5.62": "2018-10-24",
+  "5.5.61": "2018-07-30",
+  "5.5.60": "2018-04-20",
+  "5.5.59": "2018-01-18",
+  "5.5.58": "2017-10-17",
+  "5.5.57": "2017-07-18",
+  "5.5.56": "2017-04-30",
+  "5.5.55": "2017-04-11",
+  "5.5.54": "2016-12-22",
+  "5.5.53": "2016-10-14",
+  "5.5.52": "2016-09-12",
+  "5.5.51": "2016-08-09",
+  "5.5.50": "2016-06-16",
+  "5.5.49": "2016-04-20",
+  "5.5.48": "2016-02-10",
+  "5.5.47": "2015-12-09",
+  "5.5.46": "2015-10-09",
+  "5.5.45": "2015-08-04",
+  "5.5.44": "2015-06-09",
+  "5.5.43": "2015-04-29",
+  "5.5.42": "2015-02-13",
+  "5.5.41": "2014-12-19",
+  "5.5.40": "2014-10-08",
+  "5.5.39": "2014-08-03",
+  "5.5.38": "2014-06-05",
+  "5.5.37": "2014-04-15",
+  "5.5.36": "2014-02-22",
+  "5.5.35": "2014-01-28",
+  "5.5.34": "2013-11-18",
+  "5.5.33": "2013-09-16",
+  "5.5.32": "2013-07-17",
+  "5.5.31": "2013-05-21",
+  "5.5.30": "2013-03-11",
+  "5.5.29": "2013-01-29",
+  "5.5.29": "2013-01-29",
+  "5.5.30": "2013-03-11",
+  "5.5.31": "2013-05-21",
+  "5.5.32": "2013-07-17",
+  "5.5.33": "2013-09-16",
+  "5.5.34": "2013-11-18",
+  "5.5.35": "2014-01-28",
+  "5.5.36": "2014-02-22",
+  "5.5.37": "2014-04-15",
+  "5.5.38": "2014-06-05",
+  "5.5.39": "2014-08-03",
+  "5.5.40": "2014-10-08",
+  "5.5.41": "2014-12-19",
+  "5.5.42": "2015-02-13",
+  "5.5.43": "2015-04-29",
+  "5.5.44": "2015-06-09",
+  "5.5.45": "2015-08-04",
+  "5.5.46": "2015-10-09",
+  "5.5.47": "2015-12-09",
+  "5.5.48": "2016-02-10",
+  "5.5.49": "2016-04-20",
+  "5.5.50": "2016-06-16",
+  "5.5.51": "2016-08-09",
+  "5.5.52": "2016-09-12",
+  "5.5.53": "2016-10-14",
+  "5.5.54": "2016-12-22",
+  "5.5.55": "2017-04-11",
+  "5.5.56": "2017-04-30",
+  "5.5.57": "2017-07-18",
+  "5.5.58": "2017-10-17",
+  "5.5.59": "2018-01-18",
+  "5.5.60": "2018-04-20",
+  "5.5.61": "2018-07-30",
+  "5.5.62": "2018-10-24",
+  "5.5.63": "2019-01-27",
+  "5.5.64": "2019-04-26",
+  "5.5.65": "2019-07-25",
+  "5.5.66": "2019-10-30",
+  "5.5.67": "2020-01-18",
+  "5.5.68": "2020-05-06"
+}
+
+def fetch_cycles():
+    print("Fetching cycles")
+    with urllib.request.urlopen(API) as response:
+        data = json.loads(response.read().decode("utf-8"))
+        return [ x['release_id'] for x in data['major_releases'] ]
+    return []
+
+def fetch_releases(cycles):
+    releases = {}
+
+    for cycle in cycles:
+        url = API + cycle
+        print(f"Fetching version from {url}")
```

Comment:
We usually don't leave "debug" logs in scripts (and unfortunately we do not make use of the Python logging facility). So this line must be removed.